motoko/ic-pos/README.md

```diff
@@ -148,42 +150,37 @@ The icpos canister manages the store configuration and sends notifications when
 The `--argument '(0)'` argument is used to initialize the canister with `startBlock` set to 0. This is used to tell the canister to start monitoring the ledger from block 0. When deploying to the IC mainnet, this should be set to the current block height to prevent the canister from processing old transactions.
 
 ```bash
-dfx deploy --network local icpos --argument '(0)'
+dfx deploy icpos --argument '(0)'
 ```
 
 ### Step 6: Configure the icpos canister:
 
 ic-pos uses [Courier](https://courier.com/) to send email and SMS notifications. If you want to enable notifications, you need to sign up for a Courier account and and create and API key. Then issue the following command:
 
 ```bash
-dfx canister --network local call icpos setCourierApiKey "pk_prod_..."
+dfx canister call icpos setCourierApiKey "pk_prod_..."
 ```
 
 ### Step 7: Build and run the frontend:
 
 Run npm to install dependencies and start the frontend. The frontend will be available at http://localhost:5173.
 
 ```bash
-npm install
-npm run dev
+pnpm install
+dfx deploy icpos_frontend
```

Comment:
we should set the security headers to avoid this warning when deploying the frontend
```
WARN: This project does not define a security policy for some assets.
WARN: You should define a security policy in .ic-assets.json5. For example:
WARN: [
WARN:   {
WARN:     "match": "**/*",
WARN:     "security_policy": "standard"
WARN:   }
WARN: ]
WARN: Assets without any security policy: all
WARN: To disable the policy warning, define "disable_security_policy_warning": true in .ic-assets.json5.
```